### PR TITLE
Remove incorrect information on MMDS landing page

### PR DESCRIPTION
**Pull request by bdmogal**
Build: https://builds.cask.co/browse/CDAP-URUT11

### CODE REVIEW

File: cdap-ui/app/cdap/text/text-en.yaml [hunk 1]
```diff
@@ -1175,7 +1175,7 @@ features:
         b1: Intuitive Web UI for building, training, testing and evaluating Machine Learning models.
         b2: Seamless, integrated experience from data preparation and cleansing to model building and deployment.
         b3: Horizontal scalability over your Big Data environment.
-        b4: Out of the box support for common machine learning libraries like SparkML, DLF4J and H2O, along with support for deploying custom algorithms and libraries.
+        b4: Out of the box support for common machine learning libraries like SparkML.
```
Comment (ajainarayanan) [inline cdap-ui/app/cdap/text/text-en.yaml:1178]: Should also remove the corresponding use in the js file. `/Users/ajainarayanan/Projects/cdap/cdap-ui/app/cdap/components/Experiments/ExperimentsServiceControl/index.js` ?

Comment (bdmogal) [reply cdap-ui/app/cdap/text/text-en.yaml:1178]: I haven't removed this line. Only updated it.

Comment (bdmogal) [reply cdap-ui/app/cdap/text/text-en.yaml:1178]: I haven't removed this line. Only updated it.

Comment (ajainarayanan) [reply cdap-ui/app/cdap/text/text-en.yaml:1178]: oops sorry. 👍

**Comment by ajainarayanan** [comment]
👍 LGTM

**Comment by bdmogal** [comment]
Build passed. Merging.

**Pull request merged**